### PR TITLE
fix: The "Annually" text on the landing page [Pricing Section] is not properly visible, likely due to CSS positioning issues. 

### LINKED ISSUE #133: The "Annually" text on the landing page [Pricing Section] is not properly visible, likely due to CSS positioning issues.
### Issue Description

The "Annually" text on the landing page [Pricing Section] is not properly visible, likely due to CSS positioning issues.
The "Save 2 Months" button may also require CSS adjustments to ensure proper alignment and visibility of the text.

### Steps to Reproduce

1. Add or adjust CSS rules to ensure the "Annually" text is fully displayed and legible.
2. Modify CSS for the "Save 2 Months" button to prevent it from overlapping or obscuring the text.

### Expected Behavior

The "Annually" text should be fully visible and easily readable on all supported browsers and screen sizes.
It should have sufficient spacing from surrounding elements, particularly the "Save 2 Months" button.
The text color and font size should provide adequate contrast for optimal readability.
image

### Current Behavior

The "Annually" text is partially obscured or illegible due to its proximity to the "Save 2 Months" button.
The specific issue might be (e.g., overlapping text, text color blends with background, insufficient margin).
This issue affects users browsing on (mention specific browsers or screen sizes, if applicable).
image

### Screenshots (optional)

_No response_

### Operating System [e.g., Windows 10]

Windows 11

### Browser [e.g., Chrome, Firefox]

Chrome

### Environment (locally, production, selfhosted)

Locally

### Please check the boxes that apply to this issue report.

- [X] I have searched the existing issues to make sure this is not a duplicate.
- [X] I have provided steps to reproduce the issue.
- [X] I have included relevant environment information.
- [X] I have included any relevant screenshots.
- [X] I understand that this is a voluntary contribution and that there is no guarantee of resolution.
- [X] I want to work on creating a PR for this issue if approved

### PR DESCRIPTION
**Pull request by pranay911**
## What does this PR do?

- Resolved the issue with the "Annually" text visibility on the landing page.
- Addressed potential CSS positioning issues causing the problem.

Fixes #133 
classroomio

## Type of change

- [X] Bug fix (non-breaking change which fixes an issue)
- [X] Chore (refactoring code, technical debt, workflow improvements)
- [X] Enhancement (small improvements)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change adds a new database migration
- [ ] This change requires a documentation update

## How should this be tested?

- Test A
- Test B

### Required

- [ ] Filled out the "How to test" section in this PR
- [X] Self-reviewed my own code
- [ ] Commented on my code in hard-to-understand bits
- [X] Ran `pnpm build`
- [X] Checked for warnings, there are none
- [X] Removed all `console.logs`
- [X] Merged the latest changes from main onto my branch with `git pull origin main`
- [X] My changes don't cause any responsiveness issues

### Appreciated

- [X] If a UI change was made: Added a screen recording or screenshots to this PR
- [ ] Updated the ClassroomIO Docs if changes were necessary

### CODE REVIEW

File: apps/classroomio-com/.env.example [hunk 1]
```diff
@@ -1,4 +0,0 @@
-PUBLIC_ENABLE_PRICING=1
```
Comment (rotimi-best) [inline apps/classroomio-com/.env.example]: Please restore this file

File: apps/dashboard/.env.example [hunk 2]
```diff
@@ -1,30 +0,0 @@
-# Supabase setup - Required
```
Comment (rotimi-best) [inline apps/dashboard/.env.example]: Please restore this file

**Review by rotimi-best — CHANGES_REQUESTED** [review]
Thanks a lot for this @pranay911 😃. There are a couple of changes I have requested you make. In addition:

1. Please add the link to the issue in the PR where it says **Fixes issue #**
2. Upload a video demo or a screen shot showing the new update UI on mobile, tablet and desktop.

Thanks.

File: apps/dashboard/.env.example (hunk 2 above)
Comment (pranay911) [reply apps/dashboard/.env.example]: Yes, I restored the files.

**Comment by pranay911** [comment]
Fixes issue #133

---

Please find the details of the issue [here](https://github.com/rotimi-best/classroomio/issues/133).

Additionally, I have made the following changes based on your feedback:

- [Added CSS positioning to the Anually text]

---

I have also uploaded a video demo and screenshots showcasing the new update UI on mobile, tablet, and desktop. You can view them here:
image

File: apps/dashboard/.env.example [hunk 3]
```diff
@@ -1,7 +1,7 @@
 # Supabase setup - Required
 PUBLIC_SUPABASE_PROJECT_REF=localhost
-PUBLIC_SUPABASE_URL=http://localhost:54321
-PUBLIC_SUPABASE_ANON_KEY=
```
Comment (rotimi-best) [inline apps/dashboard/.env.example]: Please can you undo the changes in this file?

Use the content here

https://github.com/rotimi-best/classroomio/blob/main/apps/dashboard/.env.example

**Review by rotimi-best — CHANGES_REQUESTED** [review]
Thanks for the improvements so far. Left you another review

File: apps/dashboard/.env.example (hunk 3 above)
Comment (pranay911) [reply apps/dashboard/.env.example]: Yes , i already did

**Review by rotimi-best — APPROVED** [review]
Thank you so much for pushing through to get this merged in 😃 🎉.

You rock, thank you for your contribution.
